main.c

```diff
@@ -0,0 +1,187 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include "tree.h"
+
+void options();
+bool checkingRotates();
+bool functionCheck();

```

Comment:
```suggestion
void options(void);
bool checkingRotates(void);
bool functionCheck(void);
```
И функции должны называться глаголами в повелительном наклонении